tests/test_commands.py

```
@@ -29,8 +45,192 @@
+    assert missing_count.missing_spectra == 0
+
+    annotator = Annotator(
+        feature_generators="ms2pip,deeplc",
+        ms2pip_model="TMT",
+        ms2pip_model_path="models",
+        ms2_tolerance=0.05,
+        calibration_set_size=0.15,
+        deeplc_retrain=True,
+        processes=2,
+        id_decoy_pattern="^DECOY_",
+        lower_score_is_better=idxml_reader.high_score_better != True,
```

Ruff (0.8.2)

Avoid inequality comparisons to `True`; use `if not idxml_reader.high_score_better:` for false checks

Replace with `not idxml_reader.high_score_better`

(E712)